tests/Unit/Evolution/Systems/ScalarAdvection/FiniteDifference/Test_MonotisedCentral.cpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <memory>
+
+#include "Evolution/Systems/ScalarAdvection/FiniteDifference/MonotisedCentral.hpp"
+#include "Evolution/Systems/ScalarAdvection/FiniteDifference/Reconstructor.hpp"
+#include "Evolution/Systems/ScalarAdvection/FiniteDifference/Tags.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+
+namespace {
+template <size_t Dim>
+void test_create() noexcept {
+  const auto mc =
+      TestHelpers::test_creation<ScalarAdvection::fd::MonotisedCentral<Dim>>(
+          "");
+  CHECK(mc == ScalarAdvection::fd::MonotisedCentral<Dim>());
+}
+
+template <size_t Dim>
+void test_serialize() noexcept {
+  ScalarAdvection::fd::MonotisedCentral<Dim> mc;
+  test_serialization(mc);
+}
+
+template <size_t Dim>
+void test_move() noexcept {
+  ScalarAdvection::fd::MonotisedCentral<Dim> mc;
+  ScalarAdvection::fd::MonotisedCentral<Dim> mc_copy;
+  test_move_semantics(std::move(mc), mc_copy);  //  NOLINT
+}
+
+template <size_t Dim>
+void test_derived() {
+  const ScalarAdvection::fd::MonotisedCentral<Dim> mc_recons{};
+  const auto mc_from_options_base = TestHelpers::test_factory_creation<
+      ScalarAdvection::fd::Reconstructor<Dim>,
+      ScalarAdvection::fd::OptionTags::Reconstructor<Dim>>(
+      "MonotisedCentral:\n");
+  auto* const mc_from_options =
+      dynamic_cast<const ScalarAdvection::fd::MonotisedCentral<Dim>*>(
+          mc_from_options_base.get());
+  REQUIRE(mc_from_options != nullptr);
+  CHECK(*mc_from_options == mc_recons);
+}
+
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.ScalarAdvection.Fd.MonotisedCentral",
```

Comment:
I'm not seeing anything that actually tests the reconstruction. Did you not write a test for it?